Cargo.toml

```diff
@@ -11,6 +11,22 @@ members = [
     "boa_examples",
 ]
 
+[workspace.package]
+edition = "2021"
+version = "0.15.0"
+rust-version = "1.60"
```

Comment:
We might need to specify 1.64 here, since this is only available starting on Rust 1.64 (or cargo 1.64, in fact)

Reply:
Yep!